companies-manifest.js

```diff
@@ -853,6 +853,40 @@ const companies = [
       },
     },
   },
+  {
+    key: 'https://wazirx.com/',
+    similarWebGlobalRank: 48011,
+    name: 'WazirX',
+    logo: null,
+    style: {},
+    projects: {
+      taskade: {
+        title: 'WazirX',
+        link: 'https://wazirx.com/',
+        src: '/screenshots/thumbnails/wazirx.com.png',
+        width: 1280,
+        height: 720,
+        internalUrl: 'wazirx',
+      },
+    },
+  },
+  {
+    key: 'https://www.riotgames.com/en',
+    similarWebGlobalRank: 1819,
+    name: 'Riot Games',
+    logo: null,
+    style: {},
+    projects: {
+      taskade: {
```

Comment:
The keys on the projects need to represent the project itself. In this case it should be named something alongside the lines of lolwebsite or leagueoflegends or lolweb.
It's used in the URL for its showcase page otherwise the project's showcase page won't be able to render.